Pipfile

```diff
@@ -25,6 +25,7 @@ more-itertools = "~=8.6"
 pandas = "~=1.1"
 sqlalchemy = "~=1.3"
 pymysql = "~=1.0"
+types-python-dateutil = "*"
```

Comment:
This should be moved to `dev-packages`. Easiest is to `pipenv uninstall` then `pipenv install --dev`.

Reply:
Sorry yes of course 👍 will update